src/providers/tab/tab.ts

```diff
@@ -9,26 +13,25 @@ export class TabProvider {
   activeGiftCardsPromise: Promise<GiftCard[]>;
```

Comment:
we can also remove the bitpayCardItemsPromise declaration here

Reply:
+1